libs/platform/src/lib/components/approval-flow/approval-flow.component.ts

```diff
@@ -261,6 +264,17 @@ export class ApprovalFlowComponent implements OnInit, OnDestroy {
     /** @hidden */
     ngOnDestroy(): void {
         this.subscriptions.unsubscribe();
+        this._onDestroy$.complete();
+    }
+
+    /** @hidden Listen window resize and distribute cards on column change */
+    private _listenOnResize(): void {
+        fromEvent(window, 'resize')
+            .pipe(debounceTime(60), takeUntil(this._onDestroy$))
+            .subscribe(() => {
+                this._resetCarousel();
+                this._checkCarouselStatus();
+            });
```

Comment:
Maybe it should be added in `this.subscriptions ` for unsubscribing in `ngOnDestroy`

Reply:
added in this.subscriptions